profiler/hotapp/main.go

```diff
@@ -37,6 +37,8 @@ var (
 	version = flag.String("version", "1.0.0", "service version")
 	// Skew of foo1 function over foo2, in the CPU busyloop, to simulate diff.
 	skew = flag.Int("skew", 100, "skew of foo2 over foo1: foo2 will consume skew/100 CPU time compared to foo1 (default is no skew)")
+	// Whether to run some local CPU work to increase the self metric.
+	localWork = flag.Bool("local_work", false, "whether to run some local CPU work")
```

Comment:
Remind me again why we can't do the local work every time?

Reply:
The original version of this program is used for benchmarking. We want to preserve that capability.